rbac/server/api/packs.py

```diff
@@ -140,6 +247,27 @@
 
 
 @PACKS_BP.delete("api/packs/<pack_id>")
+@doc.summary("API Endpoint to delete specified pack")
+@doc.description(
+    "API Endpoint to delete specified pack. Only pack owners and Next Admins are allowed to "
+    "delete packs in NEXT."
+)
+@doc.consumes({"pack_id": str}, location="path", required=True)
+@doc.produces(
+    {"message": str, "deleted": int, "id": str},
+    description="Successfully deleted specified pack",
+)
+@doc.response(
+    400,
+    {"code": int, "message": str},
+    description="Pack does not exist or user is not authorized "
+    "to delete this specified pack",
+)
+@doc.response(
+    401,
+    {"code": int, "message": str},
+    description="Unauthorized: Auth token was not provided",
```

Comment:
should all PRs adding Swagger decorators be using this description for 401 errors?

Reply:
Let's discuss today and just agree on what the error should look like.

Reply:
Updated descriptions